### DISCUSSION TITLE
Pyannote.audio toolkit with MFCC

### DISCUSSION THREAD

**Discussion by sumansamui** [pyannote/pyannote-audio#1680] [Development]
I have a few doubts:

1. How to use pyannote.audio setup if we want to extract MFCC, i.e., using the speech segmentation model with the MFCC feature. Is there any pre-trained model available for those settings? Or we have to train from scratch.

2. What is the impact of sample frequency on Sincnet? I know all the input audio is downsampled or upsampled to 16k.

We observed that Pyannote provides the same result for 8k and 16k versions of a WAV file in the case of SincNet architecture. Is it because of the same number of Sinc filters in the low-frequency range for both 8k and 16k.

**Comment by hbredin** [comment]
1. There is no pretrained model available relying on MFCC. You would have to train them from scratch
2. I have only ever trained 16kHz models, so I do not really have any intuitions. I guess if you want to train 8kHz models, you would would have to slightly change the kernel size and stride of the first convolutional layer of Sincnet.

**Reply by sumansamui** [reply]
Thank you Herve, for the clarification.
